make-release.sh

```diff
@@ -27,6 +27,65 @@ usage ()
   echo "Example: $0 --version 7.27.0 --trigger-release"; echo
 }
 
+verifyContainerExistsWithTimeout()
+{
+    this_containerURL=$1
+    this_timeout=$2
+    containerExists=0
+    count=1
+    (( timeout_intervals=this_timeout*3 ))
+    while [[ $count -le $timeout_intervals ]]; do # echo $count
+        sleep 20s
+        echo "       [$count/$timeout_intervals] Verify ${1} exists..." 
+        # check if the container exists
+        verifyContainerExists "$1"
+        if [[ ${containerExists} -eq 1 ]]; then break; fi
+        (( count=count+1 ))
+    done
+    # or report an error
+    if [[ ${containerExists} -eq 0 ]]; then
+        echo "[ERROR] Did not find ${1} after ${this_timeout} minutes - script must exit!"
+        exit 1;
+    fi
+}
+
+verifyContainerExists()
+{
+    this_containerURL="${1}"
+    this_image=""; this_tag=""
+    this_image=${this_containerURL#*/}
+    this_tag=${this_image##*:}
+    this_image=${this_image%%:*}
+    this_url="https://quay.io/v2/${this_image}/manifests/${this_tag}"
+    # echo $this_url
+
+    # get result=tag if tag found, result="null" if not
+    result="$(curl -sSL "${this_url}"  -H "Accept: application/vnd.docker.distribution.manifest.list.v2+json" 2>&1 || true)"
+    if [[ $(echo "$result" | jq -r '.schemaVersion' || true) == "1" ]] && [[ $(echo "$result" | jq -r '.tag' || true) == "$this_tag" ]]; then
+        echo "[INFO] Found ${this_containerURL} (tag = $this_tag)"
+        containerExists=1
+    elif [[ $(echo "$result" | jq -r '.schemaVersion' || true) == "2" ]]; then
+        arches=$(echo "$result" | jq -r '.manifests[].platform.architecture')
+        if [[ $arches ]]; then
+            echo "[INFO] Found ${this_containerURL} (arches = $arches)"
+        fi
+        containerExists=1
+    else
+        # echo "[INFO] Did not find ${this_containerURL}"
+        containerExists=0
+    fi
+}
+
+checkRequiredImagesExist()
+{
+  IMAGE_QUAY_PREFIX="quay.io/eclipse"
+  TAG=$(head -n 1 VERSION)
+  while read -r line; do
+    IMAGE_NAME=$(echo "$line" | tr -s ' ' | cut -f 1 -d ' ')
+    verifyContainerExistsWithTimeout "${IMAGE_QUAY_PREFIX}/${IMAGE_NAME}:${TAG}" 1
+  done < "${SCRIPT_DIR}"/base_images  
```

Comment:
@mkuznyetsov looks like recent [build](https://github.com/eclipse-che/che-devfile-registry/actions/runs/770647804) has failed as variable `SCRIPT_DIR` is not set.

Reply:
Missing declaration is:

```
SCRIPT_DIR=$(cd "$(dirname "$0")" || exit; pwd)
```

Should have been reported by shellcheck plugin if you're using vscode?

Reply:
Fix in https://github.com/eclipse-che/che-devfile-registry/pull/375